--- Makefile.in	2008-12-21 19:25:35.000000000 +0100
+++ Makefile.in.oden	2008-12-21 19:27:51.000000000 +0100
@@ -6,6 +6,7 @@
 CC = @CC@
 
 CFLAGS =   @CFLAGS@
+LDFLAGS =   @LDFLAGS@
 
 MATHLIB = @MATHLIB@
 
@@ -37,7 +38,7 @@
 mawk_and_test :  mawk  mawk_test  fpe_test
 
 mawk : $(O)  rexp/.done
-	$(CC) $(CFLAGS) -o mawk $(O) $(REXP_O) $(MATHLIB) 
+	$(CC) $(CFLAGS) $(LDFLAGS) -o mawk $(O) $(REXP_O) $(MATHLIB) 
 
 mawk_test :  mawk  # test that we have a sane mawk
 	@cp mawk test/mawk
